ReportUtilityBills/resources/util.rb

```diff
@@ -125,3 +125,35 @@ def self.calculate_monthly_prorate(header, month)
     return num_days_in_month.to_f / Constants.NumDaysInMonths(year)[month - 1]
   end
 end
+
+def process_usurdb(filepath)
+  require 'json'
+
+  puts 'Parsing JSON...'
+  rates = JSON.parse(File.read(filepath))
+
+  puts 'Selecting residential rates...'
+  residential_rates = []
+  rates.each do |rate|
+    next if !rate.keys.include?('_id')
+    next if !rate['_id'].keys.include?('$oid')
+    next if !rate.keys.include?('sector')
+    next if rate['sector'] != 'Residential'
+
+    residential_rates << rate
+  end
```

Comment:
Nice. Can you also incorporate skipping rates with [these keywords](https://github.com/NREL/BEopt2/blob/c84d36bfe42a087e5ff6a6cf4756b2fc31ca0460/Modeling/Utilities/get_utility_rates.py#L109-L121)?